config/core/resources/eventtype.yaml

```diff
@@ -135,6 +136,11 @@ spec:
     - name: Reason
       type: string
       jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+  - <<: *version
+    name: v1beta2
+    served: true
+    storage: false
```

Comment:
I had one doubt, why is storage `false` here and server is `true`, unlike in #5272 where it was slightly different

Reply:
@Vishal-Chdhry see here:
* https://github.com/knative/eventing/pull/5324/files#diff-cc7409beb5d3f079b67c9fba7e5dde4f5839d237761b927794577b9b1232d77dR199
* https://github.com/knative/eventing/commit/b2387dbd2ff176cb5fea79673fb0540a6e2bc898#diff-95ba4ac834854b8d9f83dc9eddf0fb917fa5d70165a04eb22693b60d91ec9bf1R66

Both PRs did a more reasonal approach, NOT bumping both fields at the same time.

Hence I did only enalbe `servred` here